lib/Display/SFML/Sfml.cpp

```diff
@@ -97,40 +139,9 @@ void Sfml::displayGame(void)
         }
     }
     for (auto text : _texts) {
-        switch(std::get<2>(text)) {
-            case Arcade::Displays::Color::WHITE:
-                _text.setFillColor(sf::Color::White);
-                break;
-            case Arcade::Displays::Color::RED:
-                _text.setFillColor(sf::Color::Red);
-                break;
-            case Arcade::Displays::Color::GREEN:
-                _text.setFillColor(sf::Color::Green);
-                break;
-            case Arcade::Displays::Color::BLUE:
-                _text.setFillColor(sf::Color::Blue);
-                break;
-            case Arcade::Displays::Color::YELLOW:
-                _text.setFillColor(sf::Color::Yellow);
-                break;
-            case Arcade::Displays::Color::MAGENTA:
-                _text.setFillColor(sf::Color::Magenta);
-                break;
-            case Arcade::Displays::Color::CYAN:
-                _text.setFillColor(sf::Color::Cyan);
-                break;
-            case Arcade::Displays::Color::BLACK:
-                _text.setFillColor(sf::Color::Black);
-                break;
-            default:
-                _text.setFillColor(sf::Color::White);
-                break;
-        }
-        _text.setFont(_font);
         _text.setString(std::get<1>(text));
-        static const std::size_t font_size = 24;
-        _text.setCharacterSize(font_size);
-        _text.setPosition(std::get<0>(text).x * font_size, std::get<0>(text).y * font_size);
+        _text.setPosition(std::get<0>(text).x * 32, std::get<0>(text).y * 32);
```

Comment:
Why change the multiplier for the position ?
You could have kept the font size.

Reply:
Rollbacked